modules/back-end/src/Infrastructure/Webhooks/WebhookSender.cs

```diff
@@ -164,6 +177,22 @@ string ToHexString(IReadOnlyCollection<byte> bytes)
         }
     }
 
+    private async Task<WebhookDelivery> PreventEmptyPayload(Guid id, string events, string? payloadTemplate = default, Dictionary<string, object>? dataObject = default)
```

Comment:
I'd prefer not to treat this as an error.

Reply:
Looks great! I was on the fence about this aspect as well, but wasn't sure how to proceed. Thanks!